File: code_combat/7_Sarven_Desert/282-Kithgard_Enchanter/kithgard_enchanter.py
def moveRight():
    hero.moveXY(hero.pos.x + 12, hero.pos.y)

def moveDown():
    hero.moveXY(hero.pos.x, hero.pos.y - 12)

def moveUp():
    hero.moveXY(hero.pos.x, hero.pos.y + 12)


moveRight()
moveDown()
moveUp()
moveUp()
moveRight()

File: projects/code_combat/7_Sarven_Desert/282-Kithgard_Enchanter/kithgardEnchanter.js
// Define your own simple movement functions.
// Define moveRight
// Note: each function should move the hero 12 meters!
function moveRight() {
    var x = hero.pos.x + 12;
    var y = hero.pos.y;
    hero.moveXY(x,y);
}

// Define moveDown
function moveDown() {
    var x = hero.pos.x;
    var y = hero.pos.y - 12;
    hero.moveXY(x, y);
}
// Define moveUp
function moveUp() {
    var x = hero.pos.x;
    var y = hero.pos.y + 12;
    hero.moveXY(x, y);
}

// Now, use those functions!
moveRight();
moveDown();
moveUp();
moveUp();
moveRight();
